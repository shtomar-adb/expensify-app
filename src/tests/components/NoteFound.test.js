import React from 'react';
import NotFound from '../../components/NotFound';
import {shallow} from 'enzyme';

test('Should render Not Found Page', () => {
    const wrapper = shallow(< NotFound />);
    expect(wrapper).toMatchSnapshot();
    
});
