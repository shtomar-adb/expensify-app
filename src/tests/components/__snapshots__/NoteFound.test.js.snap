// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Not Found Page 1`] = `
<div>
  404!
</div>
`;
